Signal/src/ViewControllers/ConversationView/Cells/OWSQuotedMessageView.m

```diff
@@ -122,7 +122,7 @@ - (BOOL)hasQuotedAttachmentThumbnailImage
 - (UIColor *)highlightColor
 {
     BOOL isQuotingSelf = [NSObject isNullableObject:self.quotedMessage.authorId equalTo:TSAccountManager.localNumber];
-    return (isQuotingSelf ? self.conversationStyle.bubbleColorOutgoingSent
+    return (isQuotingSelf ? [self.conversationStyle bubbleColorWithIsIncoming:NO]
```

Comment:
It seems more straight forward / readable to have both `incomingBubbleColor` and `outgoingBubbleColor` methods than a "mode flag" on this single method.